### PR TITLE
update sampledata command

### PR DESCRIPTION
**Pull request by cpanato**
follow up of this PR: https://github.com/mattermost/mattermost-server/pull/11769

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -1340,6 +1340,11 @@ mattermost sampledata
           -b, --bulk string                    Optional. Path to write a JSONL bulk file instead of loading into the database.
           -w, --workers int                    How many workers to run during the import. (default 2)
 
+  User Passwords
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: Propose adding this to command description, and updating description to

```
Generate sample data and populate the Mattermost database. Supported in Mattermost v4.7 and later.

The command generates one user as the System Administrator with a username ``sysadmin`` and password ``Sys@dmin-sample1``. Other users are generated following an index, e.g. with username ``user-1`` and password ``SampleUs@r-%1``.
```

Comment (jasonblais) [reply source/administration/command-line-tools.rst]: ^how is this? I wasn't sure if my description of the index is correct

@cpanato

Comment (cpanato) [reply source/administration/command-line-tools.rst]: done, sounds good to me thanks!

Comment (jasonblais) [reply source/administration/command-line-tools.rst]: @cpanato Sorry that I wasn't clear, but I intended it to be in the command description, below the title

image

Comment (cpanato) [reply source/administration/command-line-tools.rst]: fixed @jasonblais

**Comment by jasonblais** [comment]
Thanks @cpanato

**Pull request merged**